core/src/main/java/hudson/model/RSS.java

```diff
@@ -90,4 +91,14 @@ public static void doTrackback( Object it, StaplerRequest req, StaplerResponse r
 
         req.getView(Jenkins.get(),"/hudson/"+flavor+".jelly").forward(req,rsp);
     }
+
+    public static void rss(StaplerRequest req, StaplerResponse rsp, String title, String url, RunList runList) throws IOException, ServletException {
```

Comment:
Nice 2 have. Add Javadoc to newly introduced methods

Reply:
indeed!. I thought I had added before the commit :man_facepalming: